h2o-py/tests/testdir_algos/pca/pyunit_pubdev_3916_pca_hangs.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import print_function
+from builtins import range
+import sys
+sys.path.insert(1,"../../../")
+import h2o
+from tests import pyunit_utils
+from h2o.transforms.decomposition import H2OPCA
+
+
+
+def pca_car():
+  num_runs = 10
+  run_time_c = []
+  run_time_c2 = []
+
+  for run_index in range(num_runs):  # multiple runs to get an idea of run time info
+    car = h2o.upload_file(pyunit_utils.locate("smalldata/pca_test/car.arff.txt"))
+    carPCA = H2OPCA(k=car.ncols, transform="STANDARDIZE")
+    carPCA.train(x=list(range(0, car.ncols)), training_frame=car)
+    run_time_c.append(carPCA._model_json['output']['end_time']-carPCA._model_json['output']['start_time'])
+    print("PCA model training time with car.arff.txt data in ms is {0}".format(run_time_c[run_index]))
+
+    # this part will not run because it contains customer data on my personal machine.
+    # corning = h2o.upload_file(pyunit_utils.locate("smalldata/pca_test/corning_munged_v1.csv"))
```

Comment:
For now yes since I want Nidhi and Tim to run the code and check out hangings...